index.html

```diff
@@ -0,0 +1,217 @@
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>
+          </div>
+        </div>
+      </li>
+      
+<!-- Third Page -->
+      <li class="profilexp">
+        <img class="banner" src="images/Snapshoot Availability.png" alt="Availability">
+        <div class="mainfo">
+          <h2 class="tonic" id="title">Multi-Post Stories</h2>
+          <div class="sub">
+            <h3 class="canopy">CANOPY</h3>
+            <ul class="experience">
+              <li class="bed"><a href="#">Back End Dev</a></li>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>  
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>  
+          </div>
+        </div>
+      </li>
+<!-- Fourth Page -->    
+      <li class="profilexp">
+        <img class="banner" src="images/Snapshoot GymFit.svg" alt="GymFit">
+        <div class="mainfo">
+          <h2 class="tonic" id="title">Tonic</h2>
+          <div class="sub">
+            <h3 class="canopy">CANOPY</h3>
+            <ul class="experience">
+              <li class="bed"><a href="#">Back End Dev</a></li>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>
+          </div>
+        </div>
+      </li>
+ <!-- Fifth Page -->
+      <li class="profilexp">
+        <img class="banner" src="images/Snapshoot Art.svg" alt="artprinting">
+        <div class="mainfo">
+          <h2 class="tonic" id="title">Multi-Post Stories</h2>
+          <div class="sub">
+            <h3 class="canopy">CANOPY</h3>
+            <ul class="experience">
+              <li class="bed"><a href="#">Back End Dev</a></li>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>
+          </div>
```

Comment:
- [ ] Please make sure all your project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:
![image](https://user-images.githubusercontent.com/92303891/219876301-09d640e2-026a-441f-ad3f-748112f3e233.png)